scalafix/rules/src/main/scala/fix/CurriedAssert.scala

```diff
@@ -13,7 +13,7 @@ class CurriedAssert extends SemanticRule("CurriedAssert") {
   override def fix(implicit doc: SemanticDocument): Patch =
     doc.tree.collect {
       case t @ assert(Term.Apply(name, List(value, assertion))) =>
-        Patch.replaceTree(t, name + "(" + value + ")(" + assertion + ")")
```

Comment:
This has been a warning for a long time, but now that it's part of the full build, we have to fix it to keep things passing.